### PR TITLE
Code style checker

### PR DESCRIPTION
**Pull request by DmitryBarskov**

### CODE REVIEW

**Comment by DmitryBarskov** [comment]
TODO: Take config from here: https://github.com/fs/react-next-base/blob/master/.eslintrc

File: .eslintrc.json [hunk 1]
```diff
@@ -0,0 +1,27 @@
+{
+	"env": {
+		"browser": true,
```
Comment (kehtolaulu) [inline .eslintrc.json]: omg is it tabulation

File: .eslintrc.json [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+		"ecmaVersion": 12,
+		"sourceType": "module"
+	},
+	"plugins": [
+		"react"
+	],
+	"rules": {
+	},
+	"settings": {
+    "react": {
```
Comment (kehtolaulu) [inline .eslintrc.json:23]: ...

**Review by kehtolaulu — COMMENTED** [review]
fix please

**Pull request closed without merging**